bedrock/products/templates/products/vpn/landing-refresh.html

```diff
@@ -388,8 +414,14 @@ <h3 class="c-learn-more-heading">
 {% endblock %}
 
 {% block js %}
-  {% if show_cyber_security_banner %}
-    {{ js_bundle('mozilla-vpn-cyber-security-banner') }}
+  {% if show_events_banner %}
+    {{ js_bundle('mozilla-vpn-events-banner') }}
+  {% endif %}
+
+  {% if show_events_banner %}
+    {{ js_bundle('mozilla-vpn-events-banner') }}
+    {{ js_bundle('fxa_product_button') }}
+    {{ js_bundle('data_begincheckout') }}
```

Comment:
Issue: you can remove these two bundles as they are already included in the main landing page?

Reply:
Shouldn't those two bundles (in your first comment) be loaded in the landing refresh as well? From what I see, this page is not inheriting the bundles from the OG landing page?